Source/Request.swift

```diff
@@ -781,7 +781,7 @@ open class DataRequest: Request {
 
             let result = validation(self.request, response, self.data)
 
-            result.withError { self.error = $0 }
+            if case .failure(let error) = result { self.error = error }
```

Comment:
Why not keep `withError` (and `withValue`) as extensions?

Reply:
We could, but we already have a Swift construct that is essentially the equivalent. I was trying to remove all the extensions that we could replace with either common Swift API or Result API. I'm not opposed to bringing it back, but I would advise against making it a public API if we do bring it back.

Reply:
We decided in Slack to remove them for now.

Reply:
We decided to use the Swift conventions for now and remove the `withValue` and `withError` APIs.